include/vara/Feature/FeatureModelTransaction.h

```diff
@@ -365,6 +414,15 @@ class RemoveFeatureFromModel : public FeatureModelModification {
       }
     }
 
+    while (!F->constraints().empty()) {
```

Comment:
If I understand this correctly, we go to the most upper constraint and delete it. You can we prevent that this delete much more than was intended? Not sure if a edge case can happen here, where we remove to much.

Reply:
This code should: 

1. Pick a constraint of the feature
2. Traverse upwards to get the constraint's expression tree root
3. Remove the whole Constraint from the model, this will also remove it from all other related features
4. Continue with the next constraint

Reply:
Ok, sounds ok to me. Could you please create an issue for that an reference it here with an TODO, so we come back to this later :)